Object security other than COSE is in scope of Secure Channel mechanisms (#14)
ietf-rats-wg/draft-ietf-rats-uccs: draft-ietf-rats-uccs.md

  procedural methods, or a combination thereof"

  For the purposes of the present document, we focus on a protected communication
  channel used for conveyance that can ensure the same qualities as CWT without

>   the COSE protection. For example, encrypted conveyance via PCIe, a TLS tunnel,

Author: ```suggestion
  the COSE protection. Examples include, encrypted conveyance via PCIe, a TLS tunnel,
```

>   the COSE protection. Examples include, encrypted conveyance via PCIe, a TLS tunnel,

Author: ```suggestion
  the COSE protection. Examples include encrypted conveyance via PCIe, a TLS tunnel,
```

>   or other object security than COSE, such as CMS or X.509 v3 certificate.

Reviewer: possibly missing plural

```suggestion
  or other object security than COSE, such as CMS or X.509 v3 certificates.
```
